### ISSUE TITLE
Onnx Loader

### ISSUE THREAD

**Issue by nilavghosh** [kartik4949/AutoDeploy#11]
Add onnx loader into the autodeploy
```
import numpy as np
import onnxruntime as ort
sess = ort.InferenceSession('horse_zebra.onnx')

input_name = sess.get_inputs()[0].name
label_name = sess.get_outputs()[0].name
pred_onx = sess.run(
    [label_name], {input_name: np.zeros((1, 3, 224,224), dtype=np.float32)})[0]
print(pred_onx)
```

**Comment by nilavghosh** [comment]
[horse_zebra.zip](https://github.com/kartik4949/AutoDeploy/files/7072281/horse_zebra.zip)

**Comment by nilavghosh** [comment]
FastAPI input array

```
requests.post('/predict', json = {'input' : json.dumps(im.tolist())})
class InputIn(BaseModel):
    input : str

@app.post('/')
asyc def predict(inp : InputIn):
    ...
    inp = np.array(json.loads(inp.input))
    sess.run...()

```

**Comment by kartik4949** [comment]
solved #12
